src/parallel/ocbbs.cpp

```diff
@@ -357,6 +357,7 @@ static Object** upkpyobj(void* v) {
     std::vector<char> s = bbs->upkpickle();
     assert(neuron::python::methods.pickle2po);
     Object* po = neuron::python::methods.pickle2po(s);
+    --po->refcount;
```

Comment:
Here and in three following places, we decrement the refcount to 0 (without freeing the PythonObject but the next statement increments it back to 1. An alternative is to have ```pickle2po``` return the Object* with a refcount of 0 instead of 1. I slightly prefer dealing with the recount issue individually in these 4 places where ```pickle2po``` is used.

Reply:
I believe refactoring would be a large project.  The fix here (in the four places) seems simple but it took me several weeks to actually settle on those being the right places to decrement the refcount and a long time of nervousness to convince my self I wasn't introducing a bug. The  address sanitizer was helpful in the latter regard.